packages/dashboard/src/CdcDashboard.tsx

```diff
@@ -100,21 +189,22 @@ const VisualizationsPanel = ({ loadConfig, config }) => (
       <Widget addVisualization={() => addVisualization('filter-dropdowns', '')} type='filter-dropdowns' />
     </div>
     <span className='subheading-3'>Advanced</span>
-    <AdvancedEditor loadConfig={loadConfig} state={config} />
+    <AdvancedEditor loadConfig={loadConfig} state={config} convertStateToConfig={undefined} />
   </div>
 )
 
 export default function CdcDashboard({ configUrl = '', config: configObj = undefined, isEditor = false, isDebug = false, setConfig: setParentConfig }) {
-  const [config, setConfig] = useState(configObj ?? {})
-  const [data, setData] = useState([])
-  const [filteredData, setFilteredData] = useState()
-  const [loading, setLoading] = useState(true)
+  const [config, setConfig] = useState<Config | null>(configObj ?? null)
```

Comment:
@mpallansch @adamdoe setConfig is defined twice... as a prop and as a setState function...